batch-import/post_content.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# POST a single video to a Box for segmentation and analysis
+# This requires that
+# 1. You have a Camio Box running and you can access it over the network
```

Comment:
it doesn't have to be on the local network though, some people were asking to run it on AWS in which case it would just have an external IP